src/test/java/de/fraunhofer/aisec/codyze/crymlin/CLILoadTest.kt

```diff
@@ -73,8 +73,8 @@ internal class CLILoadTest {
             Configuration.initConfig(
                 null,
                 "-c",
-                "--passes=de.fraunhofer.aisec.cpg.passes.EdgeCachePass," +
-                    "de.fraunhofer.aisec.cpg.passes.FilenameMapper," +
+                "--passes=de.fraunhofer.aisec.cpg.passes.EdgeCachePass:" +
+                    "de.fraunhofer.aisec.cpg.passes.FilenameMapper:" +
```

Comment:
This won't work on Windows because you're using the UNIX specific path separator. You should use the system property `path.separator`.